assets/js/modules/analytics/datastore/settings.test.js

```diff
@@ -421,6 +421,116 @@ describe( 'modules/analytics settings', () => {
 				expect( ( await getItem( cacheKey ) ).value ).toBeFalsy();
 			} );
 
+			it( 'does not dispatch createProperty when the `ga4Reporting` feature flag is enabled', async () => {
+				enabledFeatures.add( 'ga4Reporting' );
+
+				registry.dispatch( MODULES_ANALYTICS ).setSettings( {
+					...validSettings,
+					accountID: '12345',
+					propertyID: PROPERTY_CREATE,
+				} );
+
+				fetchMock.postOnce( gaSettingsEndpoint, ( url, opts ) => {
+					const { data } = JSON.parse( opts.body );
+					// Return the same settings passed to the API.
+					return { body: data, status: 200 };
+				} );
+
+				const result = await registry
+					.dispatch( MODULES_ANALYTICS )
+					.submitChanges();
+
+				// Ensure that the create-property request is not made.
+				expect( fetchMock ).not.toHaveFetched(
+					new RegExp(
+						'^/google-site-kit/v1/modules/analytics/data/create-property'
+					),
+					{ body: { data: { accountID: '12345' } } }
+				);
+
+				expect( result.error ).toBeFalsy();
+			} );
+
+			it( 'does not dispatch createProfile when the `ga4Reporting` feature flag is enabled', async () => {
+				enabledFeatures.add( 'ga4Reporting' );
+
+				const profileName = fixtures.createProfile.name;
+				registry.dispatch( MODULES_ANALYTICS ).setSettings( {
+					...validSettings,
+					accountID: '12345',
+					propertyID: 'UA-12345-1',
```

Comment:
These are no longer necessary to specify when the fetch assertions are simplified 👍

Reply:
Removed.